src/utils/constants.js

```diff
@@ -0,0 +1,138 @@
+export const ALERT_TYPES = [
+    'warning',
+    'error',
+    'success',
+    'information'
+];
+
+export const BADGE_TYPES = [
+    'success',
+    'warning',
+    'error'
+];
+export const LABEL_TYPES = [
+    'success',
+    'warning',
+    'error'
+];
+
+export const STATUS_TYPES = [
+    'success',
+    'warning',
+    'error',
+    'available',
+    'away',
+    'busy',
+    'offline'
+];
+
+export const BUTTON_OPTIONS = [
+    'emphasized',
+    'light',
+    'shell'
+];
+export const BUTTON_TYPES = [
+    'standard',
+    'positive',
+    'negative',
+    'medium'
+];
+
+export const FORM_MESSAGE_TYPES = [
+    'error',
+    'warning',
+    'help'
+];
+
+
+export const ICON_SIZES = [
+    's',
+    'm',
+    'l',
+    'xl'
+];
+
+export const IDENTIFIER_COLORS = [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9
+];
```

Comment:
This one seems a little silly to me.  I know `PropTypes` doesn't have a numeric range validator, but what if we create a custom one?  Something that throws an error if the number is less than 1 or greater than 9.  Seems a little easier to maintain especially if this list of colors choices ever gets big.